runtime/testy/runtime.py

```diff
@@ -139,9 +141,9 @@ def checkTimedOut(func, *args):
       testName += "_"
     setupFunc = getattr(studentCode, testName + "setup")
     mainFunc = getattr(studentCode, testName + "main")
-
```

Comment:
This whitespace noise really clutters diffs. I think it's a good habit to run `git diff` before every commit, and undoing accidental whitespace changes before sending anything to github.